x/lockup/keeper/msg_server.go

```diff
@@ -226,3 +226,24 @@ func (server msgServer) ForceUnlock(goCtx context.Context, msg *types.MsgForceUn
 
 	return &types.MsgForceUnlockResponse{Success: true}, nil
 }
+
+func (server msgServer) SetRewardReceiverAddress(goCtx context.Context, msg *types.MsgSetRewardReceiverAddress) (*types.MsgSetRewardReceiverAddressResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	owner, err := sdk.AccAddressFromBech32(msg.Owner)
+	if err != nil {
+		return nil, err
+	}
+
+	newRewardRecepient, err := sdk.AccAddressFromBech32(msg.Owner)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
This needs to be changed from msg.Owner to msg.RewardReceiverAddress

Please also add a test case that catches this as well

Reply:
Added test case in https://github.com/osmosis-labs/osmosis/pull/5281/commits/2d6aa3066c634f7dadc3feea7f9ce4c85dc14ae9